File: from-3301-to-3400/3392-count-subarrays-of-length-three-with-a-condition/solution.go
func countSubarrays(nums []int) int {
    result := 0
    for i := 1; i < len(nums) - 1; i ++ {
        if (nums[i-1] + nums[i+1]) * 2 == nums[i] {
            result += 1
        }
    }
    return result
}

File: from-3301-to-3400/3392-count-subarrays-of-length-three-with-a-condition/solution.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        result = 0
        for i in range(1, len(nums) - 1):
            if (nums[i - 1] + nums[i + 1]) * 2 == nums[i]:
                result += 1
        return result